workspace_integration_test.go

```diff
@@ -671,12 +673,13 @@ func TestWorkspacesUpdate(t *testing.T) {
 
 	t.Run("when updating a subset of values", func(t *testing.T) {
```

Comment:
We'll need to upgrade the organization for this test to pass, so adding something like this:

```
upgradeOrganizationSubscription(t, client, orgTest)
```

On or around line 670 would work.